afteraction-share.html

```diff
@@ -200,6 +200,44 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 		</div>
 	</div>
 	{% endif %}
+
+	{% if page.custom_fields.share_whatsapp_message %}
+		<script>
+		var whatsappMsg = '{{ page.custom_fields.share_whatsapp_message }} ';
+		</script>
+	{% else %}
+		<script>
+		var whatsappMsg = '{{ page.followup.share_title_value }} ';
+		</script>
+	{% endif %}
+
+	<script>
+		jQuery(document).ready(function($){
+			$('.whatsapp-share').on('click', function(e) {
+				e.preventDefault();
+				// Set Whatsapp Share URL based on response from API
+				var getShareURL = '/share/link?type=ot&action_id={{action.id}}&short=1&page_name={{page.name}}';
+				$.ajax({
+					method: 'GET',
+					url: getShareURL,
+				}).done(function(shareURL) {
+					var currWhatsappHref = $(e.target).attr('href');
```

Comment:
this currWhatsappHref var doesn't seem to be actually utilized?

Reply:
removed it!